basic/exercise5/script5.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+function firstSunday (yearX, yearY){
```

Comment:
Please rename it to something more meaningful - for example startYear and endYeara